src/components/provider/show/provider-show.js

```diff
@@ -99,52 +98,19 @@ class ProviderShow extends Component {
     return toasts;
   }
 
-  getActionMenu = () => {
-    const {
-      stripes,
-      onEdit,
-    } = this.props;
-
-    const hasEditPermission = stripes.hasPerm('ui-eholdings.records.edit');
-
-    if (!hasEditPermission) return null;
-
-    return ({ onToggle }) => (
-      <Button
-        buttonStyle="dropdownItem fullWidth"
-        onClick={() => {
-          onToggle();
-          onEdit();
-        }}
-      >
-        <FormattedMessage id="ui-eholdings.actionMenu.edit" />
-      </Button>
-    );
-  }
-
   renderLastMenu() {
-    const {
-      model: { name },
-      onEdit,
-    } = this.props;
+    const { onEdit } = this.props;
 
     return (
       <IfPermission perm="ui-eholdings.records.edit">
-        <FormattedMessage
-          id="ui-eholdings.label.editLink"
-          values={{
-            name,
-          }}
+        <Button
+          data-test-eholdings-provider-edit-link
```

Comment:
Please, add a test for this button in `provider-show-test.js`
i.e.
when `data-test-eholdings-provider-edit-link` is clicked
`this.location.pathname` should equal `/eholdings/providers/${id}/edit`